LICENSES/README.md

```diff
@@ -32,7 +32,7 @@ The following exceptions apply:
  * some sources under src/udev/ are licensed under **GPL-2.0-or-later**,
    so all udev programs (`systemd-udevd`, `udevadm`, and the udev builtins
    and test programs) are also distributed under **GPL-2.0-or-later**.
- * the header files contained in src/basic/linux/ are copied
+ * the header files contained in src/basic/linux/linux are copied
```

Comment:
src/basic/linux/linux sounds ugly...
How about src/include/linux/ ?

Please suffix with slash for directory.

Reply:
I'd like to keep the directory inside of src/basic, so I'll use src/basic/include/linux